chemprop/scripts/check_and_install_cuik_molmaker.py

```diff
@@ -0,0 +1,156 @@
+import os
+import subprocess
+import sys
+
+import rdkit
+import requests
+import torch
+
+# Check that conda is installed
+try:
+    conda_version = subprocess.check_output(["conda", "--version"], text=True).strip()
+    print(f"Conda is installed and accessible. Conda version: {conda_version}")
+except (subprocess.CalledProcessError, FileNotFoundError):
+    print("Conda is not installed or not in your system's PATH.")
+
+conda_prefix = os.getenv("CONDA_PREFIX")
+
+rdkit_so_file = os.path.join(conda_prefix, "lib", "libRDKitGraphMol.so")
+print(f"RDKit shared object file: {rdkit_so_file}")
+if not os.path.exists(rdkit_so_file):
+    print(
+        "RDKit is not installed. Please install RDKit from conda and try again."
+        "You can install RDKit with the following command: conda install -c conda-forge rdkit"
+    )
+    exit(1)
```

Comment:
RDKit should be installed if chemprop is installed, so I'd suggest adding:

```suggestion
rdkit_so_file = os.path.join(conda_prefix, "lib", "libRDKitGraphMol.so")
print(f"RDKit shared object file: {rdkit_so_file}")
if not os.path.exists(rdkit_so_file):
    print(
        f"Could not find RDKit SO file {rdkit_so_file}"
        "This probably means RDKit and/or Chemprop is not installed. Please install RDKit/Chemprop from conda and try again."
        "You can install these with the following command: conda install 'conda-forge::rdkit' 'conda-forge::chemprop'"
    )
    exit(1)
```

Will this work with `pip`? I believe `pip` is smart enough these days to install things into the `conda` `lib` directory, though maybe not.

Reply:
I doubled checked this. So if install `chemprop` from `pip`, should (1) `pip uninstall rdkit` and (2) `conda install conda-forge::rdkit`. Now, `cuik-molmaker` will work with this conda installed `rdkit`.